packages/neo4j-driver/src/index.js

```diff
@@ -305,6 +305,26 @@ function driver (url, authToken, config = {}) {
   }
 }
 
+/**
+ * Method which veirfies if the driver is server is reachable.
```

Comment:
If you agree with the suggested doc change earlier, it'd need to be changed here as well